tbot_contrib/gpio.py

```diff
@@ -0,0 +1,56 @@
+import tbot
+from tbot.machine import linux
+
+
+class Gpio:
+    def __init__(self, host: linux.LinuxShell, gpio_number: int):
+
+        self.host = host
+        self._gpio_number = str(gpio_number)
```

Comment:
I'd make this a 'public' property so downstream code could access the value.  Also, please keep it the integer number and only convert to a string when sending the commands:

```suggestion
        self.gpio_number = gpio_number
```